src/vs/workbench/browser/actions/workspaceCommands.ts

```diff
@@ -75,6 +77,41 @@ CommandsRegistry.registerCommand({
 	}
 });
 
+CommandsRegistry.registerCommand({
```

Comment:
Please move this to a folder under `src/sql`. Every addition under src/vs requires a {{ SQL Carbon Edit }} tag and makes merges harder.

Reply:
Moved it to notebook.contributions as per our conversation offline.